Database/SQLite/Simple.hs

```diff
@@ -307,6 +308,17 @@ execute conn template qs =
   withStatementParams conn template qs $ \(Statement stmt) ->
     void . Base.step $ stmt
 
+-- | Execute a multi-row @INSERT@, @UPDATE@, or other SQL query that is not
+-- expected to return results.
+--
+-- Throws 'FormatError' if the query could not be formatted correctly.
+executeMany :: ToRow q => Connection -> Query -> [q] -> IO ()
+executeMany conn template rows = withStatement conn template $ \stmt -> do
+    let Statement stmt' = stmt
+    forM_ rows $ \row ->
+        withBind stmt row
```

Comment:
Somehow `row` throws me off here.  These are statement params while a row feels more like a db row.

Maybe instead of `rows` and `row`, maybe something like `paramRows` and `params`. I don't know, doesn't feel super great either.  Postgresql-simple just uses `qs` but I don't like that better.